string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,9 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  var words = string.split(' ');
+  var capWords = '';
+  for (var i = 0; i < words.length; i++) {
+    capWords += words[i][0].toUpperCase() + words[i].slice(1).toLowerCase() + ' ';
+  }
+  return capWords.slice(0, capWords.length - 1);
```

Comment:
Instead of concatenation, why not create an array to hold the capitalized words, then `join` them after the loop ends?